hieradata/common.yaml

```diff
@@ -30,6 +30,7 @@ psick::firstrun::windows_reboot: false
 psick::pre::linux_classes:
   aws-sdk: '::psick::aws::sdk'
   puppet: psick::puppet
+  puppet_tp: psick::puppet::tp
```

Comment:
This is redundant, psick::puppet::tp is already included by default in https://github.com/example42/puppet-psick/blob/master/manifests/puppet.pp#L3